src/viewer/clients/loadReplay.ts

```diff
@@ -1,8 +1,8 @@
 import { ReplayData } from "../../models/ReplayData"
 import { ReplayMetadata } from "../../models/ReplayMetadata"
 
-const fetchByURL = (url: string) =>
-  fetch(url, {
+const fetchByURL = (url: string, local?: boolean) =>
```

Comment:
This file changes were just to make fewer network calls while testing.

Reply:
Maybe we should make it a prop that is passed in by the host?
That way we can change it as needed in the future?
Especially options as we use it in more and more different places.

Reply:
There's an `isDevelopment` utility here:
https://github.com/SaltieRL/WebReplayViewer/blob/master/src/utils/isDevelopment.ts

Reply:
Get's a bit tricky when you are in dev and want to load a different replay, instead of just changing **local** variable to **false** or not passing it at all:
- you would need to stop the webpack server
- run webpack-dev-server manually without "--mode development" argument